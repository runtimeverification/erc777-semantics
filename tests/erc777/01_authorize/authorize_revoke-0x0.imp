test.preamble();

// authorize and revoke 0x0 - both should revert.

test.authorizeOperator(0);
test.revokeOperator(0);
