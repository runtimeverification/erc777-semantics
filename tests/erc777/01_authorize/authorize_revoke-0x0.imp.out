7 is authorizing 0 as an operator
REVERT
7 is revoking operator 0
REVERT
